sentry/src/main/java/io/sentry/event/Event.java

```diff
@@ -264,9 +264,15 @@ void setEnvironment(String environment) {
         this.environment = environment;
     }
 
+    //CHECKSTYLE.OFF: JavadocMethod
     public Map<String, Object> getExtra() {
+        if (extra == null) {
```

Comment:
nit: we should add a comment indicating in which cases extra can be null (because it's generally an unexpected case).